cms/src/main/java/uk/nhs/digital/apispecs/model/SpecificationSyncData.java

```diff
@@ -13,70 +13,65 @@ public class SpecificationSyncData {
 
     private static final Pattern VERSION_FIELD_PATTERN = Pattern.compile("\"version\"\\s*:\\s*\"[^\"]+\"");
 
-    private final ApiSpecificationDocument localSpec;
-    private final OpenApiSpecification remoteSpec;
-    private final ApiSpecificationImportMetadata.Item localSpecMetadataItem;
-    private String html;
+    private final String specificationId;
+    private final String apiSpecJcrHandleNodeId;
+    private final String draftSpecJcrPath;
+    private ApiSpecificationDocument localSpec;
+    private OpenApiSpecification remoteSpec;
+    private ApiSpecificationImportMetadata.Item localSpecMetadataItem;
     private Exception error;
     private boolean published;
     private boolean eligible;
     private boolean skipped;
 
     private SpecificationSyncData(
-        final ApiSpecificationDocument localSpec,
-        final OpenApiSpecification remoteSpec,
-        final ApiSpecificationImportMetadata.Item localSpecMetadataItem
+        final String specificationId,
+        final String apiSpecJcrHandleNodeId,
+        final String draftSpecJcrPath
     ) {
-        this.localSpec = localSpec;
-        this.remoteSpec = remoteSpec;
-        this.localSpecMetadataItem = localSpecMetadataItem;
+        this.specificationId = specificationId;
+        this.apiSpecJcrHandleNodeId = apiSpecJcrHandleNodeId;
+        this.draftSpecJcrPath = draftSpecJcrPath;
     }
 
     public static SpecificationSyncData with(
-        final ApiSpecificationDocument localSpec,
-        final OpenApiSpecification remoteSpec,
-        final ApiSpecificationImportMetadata.Item localSpecMetadataItem
+        final String specificationId,
+        final String apiSpecJcrHandleNodeId,
+        final String draftSpecJcrPath
     ) {
         return new SpecificationSyncData(
-            localSpec,
-            remoteSpec,
-            localSpecMetadataItem
+            specificationId,
+            apiSpecJcrHandleNodeId,
+            draftSpecJcrPath
         );
     }
 
-    private boolean specContentDiffersIgnoringVersion(final String left, final String right) {
+    public boolean specContentDiffers() {
```

Comment:
should this function keep the original name if it's still ignoring versions?

Reply:
Yee good spot I've changed it back to the original name